exercises/concept-wip/arys-amazing-lasagna/.docs/instructions.md

```diff
@@ -0,0 +1,43 @@
+# Instructions
```

Comment:
The instructions were clear and I was able to solve the exercise following them without having to cheat or use hints 👍